Libplanet/Store/BaseStore.cs

```diff
@@ -68,32 +68,16 @@ ISet<TxId> txids
         /// <inheritdoc cref="IStore.IterateBlockHashes()"/>
         public abstract IEnumerable<BlockHash> IterateBlockHashes();
 
-        /// <inheritdoc cref="IStore.GetBlock{T}(BlockHash)"/>
-        public Block<T> GetBlock<T>(BlockHash blockHash)
+        /// <inheritdoc cref="IStore.GetBlock{T}"/>
+        public Block<T> GetBlock<T>(HashAlgorithmGetter hashAlgorithmGetter, BlockHash blockHash)
```

Comment:
Do we need `hashAlgorithmGetter` here? (there is no usage in this method...)

Reply:
It's actually needed, but I omitted passing it to the `BlockDigest`.  It's now fixed: 8455f601fea9ac362463fab8e6a1bdbba9ed1e2a.

Reply:
Also took me a while *why* this is needed. 😕

Reply:
It's because `IStore`'s some methods require `Block<T>`, which is too rich for this low level layer.  This should be fixed anyway in the future.

Reply:
Yeah, I understand this is necessary under the current design/implementation. It was half rhetorical, and only half serious to leave some paper trail, so that we have a shared elevated concern for the issue. 😛